fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1762,4 +1762,9 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         key = appContext.getPreferenceKey(R.string.pref_key_growth_early_search),
         default = false,
     )
+
+    /**
+     * Indicates if the new Search settings UI is enabled.
+     */
+    var enableNewSearchSettingsUI: Boolean = showUnifiedSearchFeature && FeatureFlags.unifiedSearchSettings
```

Comment:
nit: would it make sense to use the `unifiedSearch` language here as well to match the feature flag naming?